stories/slider/slider.stories.js

```diff
@@ -0,0 +1,146 @@
+export default {
+    title: 'Components/Slider',
+    parameters: {
+        description: `A slider is a control that enables the user to adjust single values within a specified numerical range. A range slider is a user interface control that enables the user to select a value range within a predefined numerical
+interval.`,
+        components: ['slider']
+    },
+    tags: ['f3', 'theme']
+};
+
+/**
+ * Basic slider
+ */
+
+export const basic = () => `
+<div class="slider-container">
+    <div class="fd-slider" tabindex="0" role="slider" aria-label="slider" aria-valuemin="1" aria-valuemax="100" aria-valuenow="50">
+        <div class="fd-slider__inner">
+            <div class="fd-slider__track">
+                <div class="fd-slider__track-range" style="width: 50%;"></div>
+                <div class="fd-slider__handle" style="left: 50%;"></div>
+            </div>
+        </div>
+    </div>
```

Comment:
The handle itself should contain the `slider` attributes as it will be the element to receive focus.

```suggestion
    <div class="fd-slider">
        <div class="fd-slider__inner">
            <div class="fd-slider__track">
                <div class="fd-slider__track-range" style="width: 50%;"></div>
                <div class="fd-slider__handle" tabindex="0" role="slider" aria-label="slider" aria-valuemin="1" aria-valuemax="100" aria-valuenow="50" style="left: 50%;"></div>
            </div>
        </div>
    </div>
```

It's fine that the role is not on the main `fd-slider` element; notice in [this example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role#Example_2_Text_Values) how the "slider" class is used as an outer container but the handle receives the actual slider role.

Reply:
Thanks! I moved ARIA attributes to handle elements.

However, it seems to me that it's different for other libraries, for instance, Angular Material Slider has ARIA attributes on the host container and ng5-slider on handle. Anyway, let's stick to official guidelines from MDN.

Reply:
It's definitely possible to keep the attributes on the host container given appropriate CSS and proper focus management, though it's a bit harder, especially with multiple handles.